default_protocol/genome.json

```diff
@@ -1,11 +1,46 @@
 {
-	"label": "default",
-	"length": 1000,
+	"label": "SARS-CoV-2",
```

Comment:
should we override default protocol?

Reply:
I think this is related to my comment regarding custom configuration. This is content is readily available in nanocrop. Users should not fetch rampart with our custom stuff instead of default example that is related to rampart documentation.